src/applications/vaos/services/appointment/transformers.js

```diff
@@ -381,6 +394,21 @@ function setContained(appt) {
 
       return null;
     }
+    case APPOINTMENT_TYPES.ccRequest: {
+      return appt.ccAppointmentRequest.preferredProviders.map(provider => {
+        return {
+          actor: {
+            name: provider.practiceName,
```

Comment:
We also have a provider first and last name that we need to put somewhere. Not sure if you have any thoughts on where that should go @narin. I'm inclined to just stick it in here for now, like `providerName`.

The contained structures for CC requests and appointments don't really correspond to anything in the FHIR spec. I think ideally we'd have Location and Practitioner resources in the array with the right data, but I don't want to do all that work in this PR.

Reply:
The 'Practitioner' field looks like a good candidate for the the provider name information.

Reply:
Done

Reply:
The `setParticipant()` function I created should already be returning an array.  I think it makes sense to put the provider first/last name here. Similar to the patient info, we can set the reference as something like `Practicioner/PRACTICTIONER_ID` for now so we can differentiate between patient and provider info.

Reply:
Done.